enterprise/installation.md

```diff
@@ -0,0 +1,122 @@
+
+![SSL Warning](/Resources/public/img/docs/enterprise/01-ssl-warning.png)
+
+Upload your SSL certificate on the next screen. SSL should work correctly from the next page. If your certificate requires intermediate certificates to be recognized by your browser and/or Composer, you can paste them into the certificate file together with your own certificate. Make sure your own certificate comes first, any intermediate certificate next and the root certificate last. If the order is off replicated will not recognize your certificate at all.
+
+![SSL Setup](/Resources/public/img/docs/enterprise/02-ssl-setup.png)
+
+On the next screen you can upload the supplied .rli license key file.
+
+![Upload License](/Resources/public/img/docs/enterprise/03-upload-license.png)
+
+Once validated, you will be asked to set a password for the management console or to protect it with a different authentication mechanism.
+
+![Security Console](/Resources/public/img/docs/enterprise/04-secure-console.png)
+
+### Server Settings
+In the management console you will have to configure a few options before you can start the Private Packagist Enterprise application.
+
+#### E-mail
+Select options for your SMTP server or use the GMail option for a simplified form if want to send email through GMail. This step is required to allow Private Packagist to send transactional email to users.
+
+![Email Settings](/Resources/public/img/docs/enterprise/05-email-settings.png)
+
+## Application Startup
+Save and apply the settings to start Private Packagist Enterprise. Leave the Setup mode enabled.
+
+![Start Application](/Resources/public/img/docs/enterprise/06-save-start.png)
+
+Once the application has started, click on the open link to access the Private Packagist Enterprise Web Interface.
+
+![Open Application](/Resources/public/img/docs/enterprise/07-started-open.png)
+
+## Configuring an HTTP Proxy
+If accessing websites outside of your network, e.g. packagist.org, requires an HTTP proxy then you can
+enable and configure this in the Console Settings under the HTTP Proxy section which you can access in
+the Replicated management console by clicking on the gear icon in the menu bar.
+
+![HTTP Proxy](/Resources/public/img/docs/enterprise/07-01-http-proxy.png)
+
+In case the HTTP proxy should not be used to access certain domains or IPs then you can configure such
+a NO_PROXY list in the Private Packagist application in the Admin Panel under Global Configuration.
+
+![No HTTP Proxy](/Resources/public/img/docs/enterprise/07-02-http-proxy-no-proxy.png)
+
+## Authentication Setup
+Within Private Packagist Enterprise you now need to set up at least one user authentication method. You have the choice between authentication with email addresses and passwords and several OAuth integrations with a third party service.
```

Comment:
integrations with third party services, it's not different integrations for the same service, so the services have to be plural too.